changelog.md

```diff
@@ -6,7 +6,7 @@ Please, add here all changes added since previous version of TIGRE (v2.1)
 ## Features
 
 - You can now select which GPUs to run the code on, either by name, or GPUid. By @genusn
-- Improved compilation, now there is no need to edit setup.py/xml files for your GPU/CUDA SDK
+- Improved Python compilation, now there is no need to edit setup.py for your GPU/CUDA SDK
```

Comment:
This seems to be rewriting backwards the changelog

Reply:
I didn't commit this file at the merger bb72dbe. Sorry. I should have done it.